tests/phpunit/tests/test-api.php

```diff
@@ -295,4 +295,237 @@ public function test_pll_update_term_error_path( $taxonomy, $language, $error_co
 		$this->assertInstanceOf( WP_Error::class, $result );
 		$this->assertSame( $error_code, $result->get_error_code() );
 	}
+
+	/**
+	 * @testWith ["page", true, false]
+	 *           ["page", false, false]
+	 *           ["post", false, false]
+	 *           ["page", true, true]
+	 *           ["page", false, true]
+	 *           ["post", false, true]
```

Comment:
Is there a need to test posts? What differences with pages for this test?

Reply:
The only difference is that the page can be tested with parent, i guess i could remove the tests with posts.